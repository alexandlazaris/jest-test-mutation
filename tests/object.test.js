const { basic } = require('../src/objects');

describe('test object validations', () => {
    let key = null;
    test('the name key should equal \`bob\`', () => {
        key = "bob";
        expect(basic.name).toEqual(key)
    });

    test('the age key should equal \`100\`', () => {
        key = 100;
        expect(basic.age).toEqual(key)
    });

    test('the object tree should match the snapshot', () => {
        expect(basic).toMatchSnapshot()
    });

    test('the object tree should match the inline snapshot', () => {
        expect(basic).toMatchInlineSnapshot(`
Object {
  "age": 100,
  "id": 123,
  "name": "bob",
}
`)
    });
    
});
