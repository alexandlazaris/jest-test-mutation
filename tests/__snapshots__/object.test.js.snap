// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test object validations the object tree should match the snapshot 1`] = `
Object {
  "age": 100,
  "id": 123,
  "name": "bob",
}
`;
